flutter_vlc_player/example/lib/multiple_tab.dart

```diff
@@ -64,11 +64,12 @@ class _MultipleTabState extends State<MultipleTab> {
   }
 
   @override
-  void dispose() {
+  void dispose() async {
+    super.dispose();
     for (var i = 0; i < controllers.length; i++) {
```

Comment:
Let's use a `for-in` loop here - we don't need to use indices here.
```suggestion
        for (final controller in controllers) {
```